### ISSUE TITLE
non-atomic concatenation expressions not transformed

### ISSUE THREAD

**Issue by ktkopone** [ikamensh/flynt#52]
EDIT: Removed .format() case I had typoed, only concat is broken.
```
#!/usr/bin/env python3

thing = 1
print('blah' + (thing - 1))
```
`flynt blah.py --transform-concats` does not detect any transforms in this file.
This may be trickly to handle correctly but would be nice to have.

**Comment by ktkopone** [comment]
May be same cause as https://github.com/ikamensh/flynt/issues/49

**Comment by ikamensh** [comment]
First part: missing parenthesis at the end breaks syntax, and flynt doesn't touch files with broken syntax.

```
~/try_c $ flynt tst.py                                                                                    22:04:49
Running flynt v.0.43.2

Flynt run has finished. Stats:

Execution time:                            0.006s
Files modified:                            1
Character count reduction:                 10 (14.08%)

Per expression type:
No old style (`%`) expressions attempted.
`.format(...)` calls attempted:            1/1 (100.0%)
F-string expressions created:              1

_-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_._-_.
Please run your tests before committing. Did flynt get a perfect conversion? give it a star at:
~ https://github.com/ikamensh/flynt ~
Thank you for using flynt. Upgrade more projects and recommend it to your colleagues!

~/try_c $ cat tst.py                                                                                      22:04:53
#!/usr/bin/env python3

thing = 1
print(f'blah{thing - 1}' )
```

**Comment by ktkopone** [comment]
Ah, PEBKAC, my bad I'll update this to only focus on the concat case (which is an actual issue, albeit probably tricky to handle as mentioned).

**Comment by ikamensh** [comment]
fixed in v0.44
